rasa/shared/constants.py

```diff
@@ -86,6 +86,10 @@
 DEFAULT_CORE_SUBDIRECTORY_NAME = "core"
 DEFAULT_NLU_SUBDIRECTORY_NAME = "nlu"
 DEFAULT_CONVERSATION_TEST_PATH = "tests"
+DEFAULT_MARKERS_PATH = "markers"
+DEFAULT_MARKERS_CONFIG_PATH = "markers/config"
+DEFAULT_MARKERS_OUTPUT_PATH = "markers/output"
+DEFAULT_MARKERS_STATS_PATH = "markers/stats"
```

Comment:
This is how I propose to structure the folder containing all the markers data (input and output). These constants can be used as defaults (but are not currently used in this issue).